provider/dynamodb.go

```diff
@@ -287,13 +295,15 @@ func (store *dynamodbOnlineStore) ImportTable(feature, variant string, valueType
 	store.logger.Infof("Checking metadata table for existing table %s\n", GetTablename(store.prefix, feature, variant))
 	_, err := store.GetFromMetadataTable(GetTablename(store.prefix, feature, variant))
 	if err == nil {
-		return "", fferr.NewDatasetAlreadyExistsError(feature, variant, fmt.Errorf("table %s already exists", GetTablename(store.prefix, feature, variant)))
+		wrapped := fferr.NewDatasetAlreadyExistsError(feature, variant, nil)
+		wrapped.AddDetail("tablename", GetTablename(store.prefix, feature, variant))
+		return "", wrapped
 	}
 
 	store.logger.Infof("Updating metadata table %s\n", GetTablename(store.prefix, feature, variant))
 	err = store.UpdateMetadataTable(GetTablename(store.prefix, feature, variant), valueType)
 	if err != nil {
-		return "", fferr.NewExecutionError(pt.DynamoDBOnline.String(), feature, variant, "FEATURE_VARIANT", err)
+		return "", fferr.NewResourceExecutionError(pt.DynamoDBOnline.String(), feature, variant, fferr.FEATURE_VARIANT, err)
```

Comment:
Should this error be inside UpdateMetadataTable?

Reply:
Good catch. It should. I missed that.